package.json

```diff
@@ -61,6 +61,7 @@
     "ncjsm": "^4.1.0",
     "node-fetch": "^2.6.1",
     "object-hash": "^2.1.1",
+    "path2": "^0.1.0",
```

Comment:
I've decided to go with the `path2` library for finding common path prefix - please let me know if you feel it might be better to instead of bringing a dependency, implement the solution as a part of the Framework.

Reply:
That's fine, it's safe dependency imo